src/Command/Cache/DoctrineResultCacheWarmerCommand.php

```diff
@@ -118,6 +143,16 @@ protected function configure(): void
 
         $this->setDescription('Warms up doctrine result cache.');
 
+        $this->addArgument(
+            self::ARGUMENT_CACHE_POOL,
+            InputArgument::OPTIONAL,
+            sprintf(
+                'Result cache pool to warmup [Possible values: %s] ',
+                implode(', ', [self::CACHE_POOL_USER, self::CACHE_POOL_LTI_INSTANCE])
```

Comment:
Don't know if it's good idea from your point of view, but IMO: it would be better to split this command into separate commands `DoctrineUserResultCacheWarmerCommand` and `DoctrineLtiInstancesResultCacheWarmerCommand` with, for example, `DoctrineAbstractResultCacheWarmerCommand` having all the shared code. As it would "increase the readability" of current class, and ease the understanding, when one command won't be responsible for warming up result cache for different entities.

Reply:
Yes I agree with you, these concerns should be separated. I'll prepare a ticket about this for the next refinement session , but for now I would leave it as it is if you are okay with it.